template/mkdocs.yml.jinja

```
@@ -0,0 +1,28 @@
+site_name: {{ project_name }}
+site_description: {{ project_description }}
+site_url: {{ documentation_url }}
+repo_url: {{ project_url }}
+repo_name: {{ project_url.split('/')[-2:] | join('/') }}
```

These are the recommended strictness settings for most sites according to MkDocs [1]:

```suggestion
site_name: {{ project_name }}
site_description: {{ project_description }}
site_url: {{ documentation_url }}
repo_url: {{ project_url }}
repo_name: {{ project_url.split('/')[-2:] | join('/') }}
{%- if typing == 'strict' %}

strict: true

validation:
  omitted_files: warn
  absolute_links: warn
  unrecognized_links: warn
  anchors: warn
{%- endif %}
```

[1] https://www.mkdocs.org/user-guide/configuration/#validation